docs/installation.rst

```diff
@@ -465,11 +465,13 @@ These platforms are built and tested for every change.
 +----------------------------------+----------------------------+---------------------+
 | Gentoo                           | 3.9                        | x86-64              |
 +----------------------------------+----------------------------+---------------------+
-| macOS 10.15 Catalina             | 3.7, 3.8, 3.9, 3.10, PyPy3 | x86-64              |
+| macOS 11 Big Sur                 | 3.7, 3.8, 3.9, 3.10, 3.11  | x86-64              |
+|                                  | PyPy3                      |                     |
 +----------------------------------+----------------------------+---------------------+
 | Ubuntu Linux 18.04 LTS (Bionic)  | 3.9                        | x86-64              |
 +----------------------------------+----------------------------+---------------------+
-| Ubuntu Linux 20.04 LTS (Focal)   | 3.7, 3.8, 3.9, 3.10, PyPy3 | x86-64              |
+| Ubuntu Linux 20.04 LTS (Focal)   | 3.7, 3.8, 3.9, 3.10, 3.11  | x86-64              |
```

Comment:
```suggestion
| Ubuntu Linux 20.04 LTS (Focal)   | 3.7, 3.8, 3.9, 3.10, 3.11, | x86-64              |
```
The two lines are joined into one when the RST is rendered - https://github.com/python-pillow/Pillow/blob/b4dab3aca301f49dd937fd29a06f1766cf138493/docs/installation.rst